### DISCUSSION TITLE
Web3Auth don't work when a web page is deployed

### DISCUSSION THREAD

**Discussion by salviega** [Web3Auth/Web3Auth#874] [Help]
I have this page deployed but when I want to sign in don't have permissions

![image](https://user-images.githubusercontent.com/90350943/201769881-10378433-fada-41b6-b26f-c37c095de57a.png)

I have my website in whitelist and it doesn't work either

![image](https://user-images.githubusercontent.com/90350943/201770221-d4d80dc5-4b80-483f-b95a-5b348d902dc1.png)

Help!? Why?

**Comment by lucassilver07** [comment]
I am getting exactly same issue.
It works in local environment perfectly but it does not work in production domains.

**Reply by dorukozerr** [reply]
I created [an issue](https://github.com/Web3Auth/web3auth-web/issues/884) for this. I'm having the same problem.

**Comment by tanguyenvn** [comment]
Can you check if the network of the project and the network used in the SDK are the same?
I suspect that the Web3auth project is on Mainnet while you're using Testnet in SDK
cc @lucassilver07 @dorukozerr @salviega

**Reply by lucassilver07** [reply]
Thank you for your feedback..
But I used testnet clientId and testnet (chainId:  0x61) in sdk.
![Screenshot 2022-11-17 at 11 07 39](https://user-images.githubusercontent.com/114839922/202345210-36de4cb1-7759-43d5-acad-348a686ad18a.png)

**Reply by dorukozerr** [reply]
I also checked it, network info's are the same.

**Comment by dorukozerr** [comment]
Projects created on testnet wont work but, mainnet plug and play works fine. There's a warning about testnet when creating verifiers, but not on plug-and-play.

**Reply by salviega** [reply]
> Projects created on testnet wont work but, mainnet plug and play works fine. There's a warning about testnet when creating verifiers, but not on plug-and-play.

In conclusion, my implemtation is fine. I need to deploy the project in mainnet for that web3auth work good, true?
